core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/StackV4Controller.java

```diff
@@ -462,36 +462,39 @@ public FlowIdentifier updatePillarConfigurationByCrn(Long workspaceId, @TenantAw
     @Override
     @CheckPermissionByAccount(action = AuthorizationResourceAction.POWERUSER_ONLY)
     public BackupV4Response backupDatabaseByName(Long workspaceId, String name, String backupLocation, String backupId, List<String> skipDatabaseNames,
-            @AccountId String accountId) {
+            @AccountId String accountId, int databaseMaxDurationInMin) {
         FlowIdentifier flowIdentifier = stackOperations.backupClusterDatabase(NameOrCrn.ofName(name),
-                restRequestThreadLocalService.getRequestedWorkspaceId(), backupLocation, backupId, true, skipDatabaseNames);
+                restRequestThreadLocalService.getRequestedWorkspaceId(), backupLocation, backupId, true, skipDatabaseNames,
+                databaseMaxDurationInMin);
         return new BackupV4Response(flowIdentifier);
     }
 
     @Override
     @InternalOnly
+    @SuppressWarnings("ParameterNumber")
```

Comment:
We have these for a reason, what's the justification for suppression?

Reply:
Hi @lnardai Thanks a lot for checking this PR! The reason Im not shorten the parameter by, for example, passing in a Class rather than multiple strings/integers is we are calling this API from Thunderhead, and from Thunderhead we are passing in multiple fields rather than a Class. If we are creating a new API or modifying this API with different types of parameters there needs much more changes in Thunderhead. Considering this is a bug and we want to fix it soon Im using the suppression to avoid too many changes in Thunderhead. Please let me know if this would work. Thank you again!

Reply:
is this fine by you @lnardai or would you like to request some changes?